cliquet/views/hello.py

```diff
@@ -20,6 +21,9 @@ def get_hello(request):
     if eos:
         data['eos'] = eos
 
+    if git and git.revision:
+        data['commit'] = git.revision
```

Comment:
Could you please add a note the endpoints docs ?

Especially, what happens if installing from pip ?